package-resources.yml

```diff
@@ -0,0 +1,58 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: secretgen-controller.carvel.dev.0.0.0
+spec:
```

Comment:
- Is there a way to get `releaseNotes: https://github.com/vmware-tanzu/carvel-secretgen-controller/releases/tag/v0.10.3` back in here?
- Also missing the `licenses: - Apache 2.0` section

Reply:
This file got automatically generated when the user do kctrl pkg init and I believe we tried to keep as minimal fields as possible in the Package CR and don't generate which are not compulsory.

@rohitagg2020 if we add it manually here does that help?

Reply:
I think we want to include the license section at a minimum. Whether that's done manually of via some flag to kctrl pkg int doesn't matter too much (I guess automation is always better though)

Reply:
Okay sure. We have figured out one bug that even if putting license section in package-resource.yml it does not reflect in final package.yml generated by kctrl. We will be resolving this from kctrl side